apps/ledger-live-mobile/src/components/RootNavigator/CustomImageNavigator.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useMemo } from "react";
+import { createStackNavigator } from "@react-navigation/stack";
+import { useTheme } from "styled-components/native";
+import { ScreenName } from "../../const";
+import { getStackNavigatorConfig } from "../../navigation/navigatorConfig";
+import Step1Cropping from "../../screens/CustomImage/Step1Crop";
+import Step2Preview from "../../screens/CustomImage/Step2Preview";
+import DebugScreen from "../../screens/CustomImage/DebugScreen";
```

Comment:
`DebugScreen` is not used it looks like ?

Reply:
indeed, it doesn't exist anymore actually